File: 01oop.cpp
#include <iostream>
using namespace std;

// Membuat kelas Mobil
class Mobil {
public:
    string warna;
    int tahun;

    void tampilkanInfo() {
        cout << "Warna: " << warna << endl;
        cout << "Tahun: " << tahun << endl;
    }
};

// Membuat objek dari kelas Mobil
int main() {
    Mobil mobilA;
    mobilA.warna = "Merah";
    mobilA.tahun = 2020;
    mobilA.tampilkanInfo();
    return 0;
}

File: 01oop.py
# Membuat kelas Mobil
class Mobil:
    def __init__(self, warna, tahun):
        self.warna = warna
        self.tahun = tahun

    def tampilkan_info(self):
        print(f"Warna: {self.warna}")
        print(f"Tahun: {self.tahun}")

# Membuat objek dari kelas Mobil
mobil_a = Mobil("Merah", 2020)
mobil_a.tampilkan_info()